cmd/harvester/cli/api.go

```diff
@@ -0,0 +1,45 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/HewlettPackard/galadriel/pkg/harvester"
+	"github.com/HewlettPackard/galadriel/pkg/harvester/config"
+	"github.com/spf13/cobra"
+)
+
+const defaultConfigPath = "conf/harvester/harvester.conf"
+
+var configPath string
+
+func NewRunCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "run",
+		Short: "Runs the Galadriel Harvester",
+		Long:  "Run this command to start the Galadriel Harvester",
+		Run: func(cmd *cobra.Command, args []string) {
+			configPath, _ := cmd.Flags().GetString("config")
+
+			HarvesterCLI.runHarvesterAPI(configPath)
+		},
+	}
+}
+
+func (c *HarvesterCli) runHarvesterAPI(configPath string) {
+	c.logger.Info("Configuring Harvester Cli")
+
+	cfg, err := config.LoadFromDisk(configPath)
+	if err != nil {
+		c.logger.Error("Configuring Harvester Cli")
```

Comment:
```suggestion
		c.logger.Error("Error loading Harvester config:", err)
		return
```
Not sure if this is going to interpolate the err message properly. Feel free to make the message using fmt.Sprintf if needed.
Note for future task: Add `Errorf`, `Infof`, etc to the logging interface